superset/views/core.py

```diff
@@ -3127,8 +3127,17 @@ class CssTemplateAsyncModelView(CssTemplateModelView):
 @app.after_request
 def apply_http_headers(response):
     """Applies the configuration's http headers to all responses"""
-    for k, v in config.get("HTTP_HEADERS").items():
+
+    # HTTP_HEADERS is deprecated, this provides backwards compatibility
+    for k, v in (
```

Comment:
check out the `{}.update()` method to merge dicts

Reply:
@mistercrunch updated!